core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/CoreDocumentSynchronizationConfig.java

```diff
@@ -228,6 +267,16 @@ void setSomePendingWrites(
     }
   }
 
+  void setPendingWritesCompleteNoDB(final BsonDocument atVersion) {
```

Comment:
[q] Is the point of this to make sync not skip over these during batch processing?

Reply:
The point of this is to not replace the doc config everytime we set pending writes– that's very slow. So we change it in memory here and do a bulk write after the pass on the namespace is done.

Reply:
cool